### PR TITLE
Execute sacura job in parallel

### PR DESCRIPTION
**Pull request by pierDipi**
Part of #596

## Proposed Changes

- Execute sacura job in parallel

### CODE REVIEW

File: test/e2e-tests.sh [hunk 1]
```diff
@@ -40,15 +39,10 @@ export_logs_continuously "kafka-broker-dispatcher" "kafka-broker-receiver" "kafk
 
 failed=false
 
-go_test_e2e -timeout=30m ./test/... || failed=true
+go_test_e2e -timeout=50m ./test/... || failed=true
```
Comment (slinkydeveloper) [inline test/e2e-tests.sh:42]: Uhhhhh are you sure of that?

Comment (pierDipi) [reply test/e2e-tests.sh]: I changed again to use a 2 step test since GH actions don't run the sacura job and reverted this timeout change.

**Pull request merged**